synfig-studio/src/gui/actionmanagers/layeractionmanager.cpp

```diff
@@ -191,7 +191,7 @@ LayerActionManager::LayerActionManager():
 {
 	action_cut_=Gtk::Action::create_with_icon_name(
 		"cut",
-		"edit-cut", _("Cu_t"), _("Cut")
```

Comment:
This is not a typo ;) It is a [mnemonic](https://en.wikipedia.org/wiki/Mnemonics_(keyboard))

Reply:
Hahaha, I didn't think it was gonna be that... sorry.
I reverted it ;-)